go/client/cmd_team_leave.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdTeamLeave struct {
+	libkb.Contextified
+	team       string
+	permanence bool
+}
+
+func newCmdTeamLeave(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "leave",
+		ArgumentHelp: "<team name> --permanence=<true|false>",
+		Usage:        "leave a team",
+		Action: func(c *cli.Context) {
+			cmd := &CmdTeamLeave{Contextified: libkb.NewContextified(g)}
+			cl.ChooseCommand(cmd, "leave", c)
+		},
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "p, permanence",
```

Comment:
can you change name to `permanent`?   Just a simpler word, that won't confuse people as much.